src/styles/components/clock.scss

```diff
@@ -0,0 +1,63 @@
+.container {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  max-width: 1200px;
+  margin: 0 auto;
+}
+
+.clock {
+  position: relative;
+  @include watch-size;
+  background: linear-gradient(
+    99deg,
+    rgb(255, 166, 0) 14.7%,
+    rgb(255, 99, 97) 73%
+  );
+  box-shadow: 3px 10px gray;
+  border-radius: 50%;
+
+  &__center {
+    position: absolute;
+    left: 50%;
+    top: 50%;
+    transform: translate(-50%, -50%);
+    @include dot-size;
+    border-radius: 50%;
+    background-color: $hand-color;
+  }
+
+  &__second {
+    position: absolute;
+    top: 20%;
+    left: 50%;
+    height: 150px;
+    width: 5px;
+    background-color: $hand-color;
+    border-radius: 50%;
+    transform-origin: center bottom;
+    animation: rotate steps(60) 60s infinite;
+  }
+
+  &__minute {
+    position: absolute;
+    top: 30%;
+    left: 50%;
+    height: 100px;
+    width: 5px;
+    background-color: $hand-color;
+    border-radius: 50%;
+    transform-origin: center bottom;
```

Comment:
Why bottom?
transform-origin: center bottom;

Why not?
transform-origin: center left;

![image](https://user-images.githubusercontent.com/33294729/173827978-359930e7-0540-49bd-a57d-c41bf2f6f1fc.png)